### PR TITLE
pass CXX_FLAGS through to gyp

### PR DESCRIPTION
**Pull request by mikemorris**
to allow setting flags like `-fPIC`

### CODE REVIEW

File: geojsonvt.gyp [hunk 1]
```diff
@@ -42,6 +42,7 @@
           '<@(variant_cflags)',
           '<@(geometry_cflags)',
           '<@(rapidjson_cflags)',
+          '${CXX_FLAGS}',
```
Comment (kkaefer) [inline geojsonvt.gyp]: Typically, the environment variable is called `CXXFLAGS` (without the underscore). Any reason this one has the underscore?

Comment (mikemorris) [inline geojsonvt.gyp]: Oops, you're right, will fix =)

This is specifically to pass through https://github.com/mapbox/mason/blob/master/scripts/geojsonvt/4.1.2-cxx11abi/script.sh#L28

**Comment by springmeyer** [comment]
Closing as invalid. The way to pass `CXXFLAGS` to make (generated by gyp) is to set them in the environment.

**Comment by kkaefer** [comment]
@springmeyer um, why? this change specifically /is/ about using the variable from the environment.

**Comment by mikemorris** [comment]
From chat:
> `CXXFLAGS` is respected automatically by make systems built with gyp so that edit is incorrect

I'll double-check this, it could be that this was just an issue with the binary, and building it from source DID respect `-fPIC` in `CXXFLAGS`...

**Pull request closed without merging**